bugs/sru-19.4.33-manual-1.txt

```diff
@@ -0,0 +1,338 @@
+=== Begin SRU Template ===
+[Impact] 
+Azure Gen2 VMs which migrate from one Azure backend server to another
+will run into a scenario where dmi data reports byte-swapped DMI product_uuid.
+
+In this scenario, cloud-init should not reprovision the system due to 
+New Instance first boot event as cloud-init will have already configured the
+VM appropriately.
+
+This fix should avoid re-provisioning when DMI product_uuid on Azure VMs
+happen to have byte-swapped ordering for the first 3 segments of the product
+UUID.
+
+[Test Case]
+
+#!/bin/sh
+set -x
+# Manually deploy on Azure using Azure CLI client.
+# Fake a byte-swapped cached instance-id and validate that old cloud-init
```

Comment:
I would suggest moving this comment out of the verification shell script. I'd also like to see a brief explanation of why we're faking this instead of testing it "properly".